src/CMakeLists.txt

```diff
@@ -160,7 +160,7 @@ install(TARGETS mgmol_src DESTINATION lib)
 add_executable(mgmol-opt main.cc)
 target_include_directories (mgmol-opt PRIVATE ${Boost_INCLUDE_DIRS})
 
-target_link_libraries(mgmol-opt mgmol_src ${link_libs}) 
```

Comment:
this change seems to create build issues on some platforms. some .o end up behind the external libraries in the link line leading to undefined functions.

Reply:
ok let's change it back then